triggers/jenkins/Jenkinsfile.yml

```diff
@@ -0,0 +1,12 @@
+pipeline:
```

Comment:
For now I would move this file to jenkins/Jenkinsfile.yml. You could leave gitbot_ci.sh where it stands if you want, you'll just have to change line 10. This is not ideal, but for now, at least the pipeline will run. We will have to create a task to improve the bit of using the right Jenkinfile.yml